awx/main/scheduler/task_manager.py

```diff
@@ -146,7 +145,7 @@ def spawn_workflow_graph_jobs(self):
         for workflow_job in self.all_tasks:
             if self.timed_out():
                 logger.warning("Workflow manager has reached time out while processing running workflows, exiting loop early")
-                self.schedule_manager.schedule()
+                WorkflowManager().schedule()
```

Comment:
elsewhere, in this same method you do `ScheduleWorkflowManager().schedule()`, and that looks like the correct pick to me.